src/kernels/errors/kernelErrorHandler.ts

```diff
@@ -474,7 +474,7 @@ function getJupyterMissingErrorMessageForCell(err: JupyterInstallError) {
     const productNames = `${ProductNames.get(Product.jupyter)} ${Common.and()} ${ProductNames.get(Product.notebook)}`;
     const moduleNames = [Product.jupyter, Product.notebook].map(translateProductToModule).join(' ');
 
-    const installerCommand = `python -m pip install ${moduleNames} -U\nor\nconda install ${moduleNames} -U`;
+    const installerCommand = `python -m pip install ${moduleNames} -U\nor\nconda install ${moduleNames} --update-deps --force-reinstall`;
```

Comment:
Not sure `--update-deps --forcereinstall` is what we want. I think it should say
```suggestion
    const installerCommand = `python -m pip install ${moduleNames} -U\nor\nconda install ${moduleNames}\nor\nconda update ${moduleNames}`;
```